app/services/ResourceIndexer.java

```diff
@@ -62,14 +75,11 @@ public ResourceIndexer(Model aDb, Writable aTargetRepo, GraphHistory aGraphHisto
    * @return The list of resources touched by the diff
    */
   public Set<String> getScope(Commit.Diff aDiff) {
-
```

Comment:
I actually like a blank line here and there to structure the code ;)

Reply:
Let's talk about that. I like empty lines / optical code structure, too, but in cases like these I prefer them to be outside the method. The reason is: otherwise the method declaration is "hanging in the middle of nowhere", you know what I mean? It is harder to capture the extent of a (method) entity. If it's be acceptable for you, let us prefer two empty lines above a method's declaration instead.